### ISSUE TITLE
React? Flux?

### ISSUE THREAD

**Issue by rickyvetter** [socialtables/WebGLVM#6]
I noticed that React is required in here. Is that what you're intending on using for the UI? Would you be cool with adding a flux-type architecture that could act as the store for both React UI elements and the actual WebGL scene? I think it would add a lot of really positive structure to the app. If so it would be better to add it sooner rather than later and I'd be happy to work on that!

**Comment by tiansijie** [comment]
yes, I was trying to use the react. You could do whatever you want is best for this app with react since you are the expert!! I would work on the camera type changing staff
